sys/Makefile

```diff
@@ -170,6 +167,12 @@ endif
 ifneq (,$(filter ng_netdev_eth,$(USEMODULE)))
     DIRS += net/link_layer/ng_netdev_eth
 endif
+ifneq (,$(filter ng_pktbuf_default,$(USEMODULE)))
+    DIRS += net/crosslayer/ng_pktbuf_default
+endif
+ifneq (,$(filter ng_pktbuf_dynamic,$(USEMODULE)))
+    DIRS += net/crosslayer/ng_pktbuf_dynamic
+endif
```

Comment:
I don't understand your system for sys/Makefile... This however is prone to conflict with other PRs that just squish the path defintitions to the end